spec/static/spec/v1/openapitransactional.yaml

```diff
@@ -92,6 +92,34 @@ tags:
         - <span style="color:red">*Files bigger than 10 MB should use compression,
           see [Compression](#section/Compression)*</span>
 
+      ## Steps to upload Assets
+
+      To upload assets files there are generally three steps involved:
+
+      <ol>
+        <li>Create new upload process</li>
+        <li>Upload parts/files</li>
+        <li>Complete upload process</li>
+      </ol>
+
+      Any file that is larger than 5 GB must be split into multiple parts. A file part must be at
+      least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+      If the file is less that 5 GB, you will only upload a single part, but must still start and complete the process as with multiple parts.
+
+      (1) Use the [create new upload](#tag/Asset-Upload-Management/operation/createAssetUpload) request to start a new upload.
+      It will return a list of urls.
+
+      (2) Use the urls to [upload asset file part](#tag/Asset-Upload-Management/operation/uploadAssetFilePart). Do this for each file part.
+
+      (3) Once all parts have been uploaded, execute the [complete the upload](#tag/Asset-Upload-Management/operation/completeMultipartUpload) request.
+      The new asset file be available to users after you have successfully completed the upload.
+
+      If any errors happen during the upload process, you can [abort the upload](#tag/Asset-Upload-Management/operation/abortMultipartUpload)
+      and restart the process.
+
+      There can only be 1 upload in progress per asset. If you wish to start a new upload you must first abort any existing upload that is in progress.
```

Comment:
I think this is a bit unclear. From my understanding it's OK to upload multiple parts concurrently but you cannot "overwrite" (start a new upload of) an asset for which upload has not completed (or been aborted).

Reply:
Esp. the flow chart could be of interest to some users

Reply:
I wasn't aware of that doc. Maybe it should be pulled into the API doc itself instead of being a separate document? If it needs to remain separate, I strongly support linking to it.

Reply:
Adapted the concurrent upload part sentence. I was also not aware of this document. I added the detailed description of the upload to the spec, as well as the diagram asset to the static files.

Reply:
I suggest replacing the document with a redirect to the API doc that has the content. Less copypasta to maintain.

Reply:
Yes, I removed the existing file as all the content is integrated in the spec. Also moved both diagram files to the static assets.